src/stores/status.ts

```diff
@@ -27,5 +27,5 @@ export const updates = readable(false, (set) => {
 
 export default statusable({
 	ping: API_BASEURL,
-	sse: STREAM_URL,
+	// sse: STREAM_URL,
```

Comment:
@Valexr why stream is now commented out?

Reply:
To save the number of sse connections in the tab.